workcraft/PlatoPlugin/src/org/workcraft/plugins/plato/gui/WriterDialog.java

```diff
@@ -210,16 +210,16 @@ private void updateLastDirUsed() {
 
     private String getDefaultComponentText() {
         String result = "module Concept where\n" + "\n" + "import Tuura.Concept.";
-        result = result + (fst ? "FSM" : "STG");
-        result = result + "\n\n" + "component a b c = \n" + "  where";
+        result += (fst ? "FSM" : "STG");
+        result += "\n\n" + "component a b c = \n" + "  where";
```

Comment:
:thought_balloon: How do you think about to use a single string literal for such  a statement?

Reply:
Yes, this is coming with the commit for #1409